docs: Prometheus database snapshot (#3348)
scality/metalk8s: docs/operation/sosreport.rst

Sosreport
=========

> sosreport utility tool is deployed automatically alongside any MetalK8s
> installation, embedding some custom MetalK8s related plugins.

Reviewer: ```suggestion
The sosreport tool is installed automatically on all MetalK8s hosts,
and embeds some custom plugins.
```

> It allows to generate reports for a machine, including logs, configurations,

Reviewer: ```suggestion
It allows to generate a report from a host, including logs, configurations,
```

containers information, etc.

> This report can then be consumed by an operator to investigate problems on a
> platform.

Reviewer: ```suggestion
This report can then be consumed by an operator, or shared with Scality support,
to investigate problems on a platform.
```

.. _sosreport-generate-report:

Generate A Report
-----------------

> To generate a report for a machine, you must have the root access.

Reviewer: ```suggestion
To generate a report for a machine, you must have root access.
```

> For example, to do a report with all containerd and metalk8s related logs and
> configurations:

Reviewer: ```suggestion
To include logs and configuration for containerd and MetalK8s components, run:
```

> .. code-block:: shell
> 
>    sosreport --batch --all-logs \
>        -o metalk8s -kmetalk8s.all=True -kmetalk8s.podlogs=True -kmetalk8s.describe=True \
>        -o containerd -kcontainerd.all=True -kcontainerd.logs=True

Reviewer: ```suggestion
.. code-block:: console

   root@your-machine # sosreport --batch --all-logs \
       -o metalk8s -kmetalk8s.all=True -kmetalk8s.podlogs=True -kmetalk8s.describe=True \
       -o containerd -kcontainerd.all=True -kcontainerd.logs=True
```

> You can retrieve the name of the generated archive in the sosreport output.

Reviewer: ```suggestion
The name of the generated archive is printed in the console output.
```

(And maybe add an example)

Plugins List
------------

> You can display the full list of available plugins and their options:

Reviewer: ```suggestion
To display the full list of available plugins and their options, run:
```
